src/lib/components/commons/ToolBar/ToolBar.types.ts

```diff
@@ -0,0 +1,15 @@
+import { EToolbarButtons, EToolbarText } from './ToolBar.consts';
+
+export type IToolBarProps = {
+  toolbar: {
+    buttons: EToolbarButtons[];
+    display: Record<EToolbarText, string | ((arg0: string) => string)>;
```

Comment:
you can move this entire type `((arg0: string) => string)` to its own named type , and use it here and in other places. e.g.:
```ts
export type IGetDisplayTextFunc = ((arg0: string) => string)
```

Reply:
see also https://github.com/shlomogcap/b-cont-react/pull/19#discussion_r1287057013

Reply:
done!